### DISCUSSION TITLE
model save to and load from a directionary?

### DISCUSSION THREAD

**Discussion by DonghanHe** [jax-ml/jax#7316] [Q&A] [answered]
Hey I dunno if I am searching wrong or what, but how do you save flax's nn.Module models? I tried jnp.save('...', frozen_dict_of_auto_encoder_params) and the loaded stuff are just ['encoder', 'decoder'] and nothing else. Is there something like a tf.saved_models.save('...')?

**Comment by lamflokas** [comment, marked as answer]
This [link](https://colab.research.google.com/github/BertrandRdp/flax/blob/master/docs/notebooks/flax_basics.ipynb#scrollTo=0eAPPwtpXYu7) shows an example on how to save and load FrozenDicts in Flax. The [flax serialiation package](https://flax.readthedocs.io/en/latest/flax.serialization.html) contains a lot of helpful functions.

**Reply by DonghanHe** [reply]
awesome, thank you
